app/questionnaire/questionnaire_manager.py

```diff
@@ -91,7 +91,7 @@ def build_state(self, item_id, answers):
             self.state.update_state(answers)
             self._conditional_display(self.state)
 
-            context = build_schema_context(metadata, self._schema.aliases, all_answers)
+            context = build_schema_context(metadata, self._schema.aliases, all_answers, request.view_args)
```

Comment:
Should we be passing these from the view as opposed to grabbing them from the request lower down?